docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
+* **Error Flags**: Indicates if the motor is overheating
+
+The frequency that this message is published at is determined by the :ref:`dronecan_support_telemetry_frequency` configuration parameter.
+
+Refer to the `uavcan.equipment.device.Temperature section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#temperature>`_ in the specification 
+for more details
+
+
+.. _dronecan_messages_raw_command:
+
+uavcan.equipment.esc.RawCommand (Data Type ID = 1030)
+######################################################
+Used to control the speed and direction of the motor. This message should be sent from the flight controller to the motors. The payload consists of a list of up to 20 values, 
+with each value interpreted as a 14 bit signed integer. Each entry in the list corresponds to a motor with the given ESC index. E.g. The first entry in the list of raw commands 
+will be read by the motor that has been assigned ESC index 0, the second entry will be read by the motor with ESC index 1, and so on. 
+
+The values represent the speed and direction to command the motor to, normalized over a range of [-8192, 8191], with -8192 being full speed in reverse and 8191 being full speed forward.
+
+Refer to the `uavcan.equipment.esc.RawCommand section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#rawcommand>`_ in the specification 
+for more details
+
+Service Requests
+*****************
+Service requests are messages sent to a specific target node from another node, and to which the sending node expects to receive a response message.
+
+uavcan.protocol.GetNodeInfo (Data Type ID = 1)
+################################################
+This request has no payload fields. The response message from the receiving node should include the status of the node as defined by the NodeStatus message, the 
+software and hardware version of the node, and the name of the node. For Vertiq modules, the name of each node is currently “iq_motion.esc”.
+
+Refer to the `uavcan.protocol.GetNodeInfo section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#getnodeinfo>`_ in the specification 
+for more details
+
+uavcan.protocol.param.GetSet (Data Type ID = 11)
+##################################################
+Used to get or set the value of a configuration parameter using either the index or the name of the parameter. The configuration parameters currently available on standard 
+Vertiq modules are listed in the :ref:`dronecan_configuration_parameters` section below. The request message should contain the index of the parameter or the name of the parameter depending 
+on how you are accessing it, and it should contain a value to set the parameter to if you are setting it. **Parameter indices are not guaranteed to remain consistent across 

```

Comment:
The request message should contain the index or name of the parameter (depending on how you are accessing it), and if performing a set operation, should contain the value to set to the parameter.